examples/sdk-examples/src/index.tsx

```diff
@@ -9,6 +9,8 @@ import "@gooddata/sdk-ui-filters/styles/css/main.css";
 import "@gooddata/sdk-ui-charts/styles/css/main.css";
 import "@gooddata/sdk-ui-pivot/styles/css/main.css";
 import "@gooddata/sdk-ui-kit/styles/css/main.css";
+import "@gooddata/sdk-ui-ext/styles/internal/css/dashboard_embedding.css";
+import "@gooddata/sdk-ui-ext/styles/internal/css/layout.css";
```

Comment:
This should not be necessary after we merge https://github.com/gooddata/gooddata-ui-sdk/pull/877 (layout.scss is imported in dashboard_embedding.scss)

Reply:
Removed the line